config/locales/simple_form.zh_TW.yml

```diff
@@ -8,6 +8,21 @@ zh-TW:
       # You can uncomment the line below if you need to overwrite the whole required html.
       # When using html, text and mark won't be used.
       # html: '<abbr title="required">*</abbr>'
+    options:
+      leave_time:
+        leave_type:
+          remote: '在家上班'
+          annual: '特休'
+          bonus: '補休'
+          personal: '事假'
+          sick: '病假'
+          fullpaid_sick: '全薪病假'
+          halfpaid_sick: '半薪病假'
+          official: '公假'
+          marriage: '婚假'
+          maternity: '產假'
+          bereavement: '喪假'
+          absent: '曠職'
     error_notification:
       default_message: "Please review the problems below:"
       leave_time:
```

Comment:
Just make a double check, `leave_time` translation under `error_notification` scope seems unnecessary, plz remove it 🙇